contracts/BadgerYieldSource.sol

```diff
@@ -4,13 +4,14 @@ pragma solidity 0.6.12;
 
 import { IYieldSource } from "@pooltogether/yield-source-interface/contracts/IYieldSource.sol";
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
 import "./IBadgerSett.sol";
 import "./IBadger.sol";
 import "hardhat/console.sol";
 
 /// @title A pooltogether yield source for badger sett
 /// @author Steffel Fenix, 0xkarl
-contract BadgerYieldSource is IYieldSource {
+contract BadgerYieldSource is IYieldSource, ReentrancyGuard {
```

Comment:
You must initialize the guard!

Reply:
Done: https://github.com/pooltogether/badger-yield-source/pull/1/commits/0975a756a97245c595b94cf3b741ea905f863bc8